run_lints.sh

```diff
@@ -2,4 +2,4 @@
 
 set -ex
 
-go vet ./...
+golangci-lint run
```

Comment:
This is assuming that `golangci-lint` is installed and that is the correct version - my understanding is that you can use `go run` here instead to help handle both of those:

```suggestion
go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1 run ./... --max-same-issues 0
```

(`--max-same-issues` is what I've got over in `osv-detector` as I feel it's better to have a lint tool show you all the results at once - feel free to remove if you disagree)

Reply:
Thanks, that makes sense.